Tools/binasciibench/binasciibench.py

```diff
@@ -0,0 +1,193 @@
+    bench_func(data, num_ops)
+
+    # Timed runs
+    times_ns = []
+    for _ in range(iterations):
+        elapsed_ns = bench_func(data, num_ops)
+        times_ns.append(elapsed_ns)
+
+    return times_ns, num_ops
+
+
+def format_throughput(bytes_per_second):
+    """Format throughput in human-readable units."""
+    if bytes_per_second >= 1_000_000_000:
+        return f"{bytes_per_second / 1_000_000_000:.2f} GB/s"
+    elif bytes_per_second >= 1_000_000:
+        return f"{bytes_per_second / 1_000_000:.2f} MB/s"
+    elif bytes_per_second >= 1_000:
+        return f"{bytes_per_second / 1_000:.2f} KB/s"
+    else:
+        return f"{bytes_per_second:.2f} B/s"
+
+
+def format_size(size):
+    """Format size in human-readable units."""
+    if size >= 1_048_576:
+        return f"{size // 1_048_576}M"
+    elif size >= 1024:
+        return f"{size // 1024}K"
+    else:
+        return str(size)
+
+
+def print_results(name, size, times_ns, num_ops, data_size):
+    """Print benchmark results."""
+    # Calculate statistics
+    times_per_op_ns = [t / num_ops for t in times_ns]
+    mean_ns = statistics.mean(times_per_op_ns)
+    stdev_ns = statistics.stdev(times_per_op_ns) if len(times_per_op_ns) > 1 else 0
+
+    # Calculate throughput
+    bytes_per_ns = data_size / mean_ns
+    bytes_per_second = bytes_per_ns * 1_000_000_000
+    throughput = format_throughput(bytes_per_second)
+
+    # Calculate coefficient of variation
+    cv = (stdev_ns / mean_ns * 100) if mean_ns > 0 else 0
+
+    size_str = format_size(size)
+    print(f"{name:<20} {size_str:>8}  {mean_ns:>12.1f} ns  "
+          f"(+/- {cv:>5.1f}%)  {throughput:>12}")
+
+
+def run_all_benchmarks(sizes):
+    """Run all benchmark variants for all sizes."""
+    print(f"binascii base64 benchmark")
+    print(f"Python: {sys.version}")
+    print(f"Target time per benchmark: {TARGET_TOTAL_TIME_S}s")
+    print()
+    print(f"{'Benchmark':<20} {'Size':>8}  {'Time/op':>15}  "
+          f"{'Variance':>10}  {'Throughput':>12}")
+    print("-" * 75)
+
+    for size in sizes:
+        # Generate test data
+        binary_data = generate_test_data(size)
+        base64_data = generate_base64_data(size)
+
+        # Benchmark encode
+        times, num_ops = calibrate_and_run(benchmark_encode, binary_data,
+                                           TARGET_TOTAL_TIME_S)
+        print_results("b2a_base64", size, times, num_ops, size)
+
+        # Benchmark decode
+        times, num_ops = calibrate_and_run(benchmark_decode, base64_data,
+                                           TARGET_TOTAL_TIME_S)
+        print_results("a2b_base64", size, times, num_ops, size)
+
+        print()
+
+
+def main():
+    parser = argparse.ArgumentParser(
+        description="Benchmark binascii base64 encoding and decoding",
+        formatter_class=argparse.RawDescriptionHelpFormatter,
+        epilog=__doc__
+    )
+    parser.add_argument(
+        "-s", "--sizes",
+        type=str,
+        default=None,
```

Comment:
```suggestion
        default=",".join(map(str, DEFAULT_SIZES)),
```

That way, we know what's the default values are (you'll need a formatter class inheriting from DefaultHelpFormatter as well)